Add `--dump-running` option to under the `revisions` sub-command

moznion opened the pull request:
to dump the running task definition on the service.

Example:

```
$ ecspresso revisions --dump-running
{
  "compatibilities": [
    "EC2",
    "FARGATE"
  ],
  "containerDefinitions": [
    ...
  ],
  ...
}
```

moznion commented:
According to [the conversation on Twitter](https://twitter.com/fujiwara/status/1631533014181347335), it would be better to implement this option under the `revisions` sub-command instead. That new option should retrieve the current running task definition of the service and dump it.
I appreciate your feedback!

fujiwara commented:
@moznion Thank you! I think this feature should be added to `revisions` command.

moznion committed:
Add `--dump-running` option to under the `revisions` sub-command

to dump the running task definition on the service.

Example:

```
$ ecspresso revisions --dump-running
{
  "compatibilities": [
    "EC2",
    "FARGATE"
  ],
  "containerDefinitions": [
    ...
  ],
  ...
}
```

Signed-off-by: moznion <moznion@mail.moznion.net>

moznion commented:
@fujiwara I replaced the implementation to dump the running task's definition on [d0044ca](https://github.com/kayac/ecspresso/pull/525/commits/d0044ca1b15535630f0317aa06305a80d532740f).

fujiwara commented:
@moznion I tried another approach. Please confirm whether #539 will resolve your issue or not.

moznion commented:
@fujiwara I'm sorry for my late reply, that seems perfect for me.Thank you so much!

fujiwara commented:
fixed by #539
